src/monitor.py

```diff
@@ -0,0 +1,75 @@
+from screeninfo import get_monitors
+
+
+class Monitor:
+    def __init__(self, name, width, height, x, y, primary, wallpaper=None):
+        self.width = width
+        self.height = height
+        self.wallpaper = wallpaper
+        self.primary = primary
+        self.x = x
+        self.y = y
+        self.name = name
+
+    def set_wallpaper(self, wallpaper):
+        self.wallpaper = wallpaper
+
+    def __str__(self):
+        return f"Monitor(name={self.name}, x={self.x}, y={self.y}, width={self.width}, height={self.height}, primary={self.primary}, wallpaper={self.wallpaper})"
+
+
+class MonitorInfo:
+    @staticmethod
+    def get_unique_monitor_count():
+        monitors = get_monitors()
```

Comment:
You can replace the `sreeninfo` by using gdk, for example:
```python
import gi
gi.require_version('Gdk', '3.0')
from gi.repository import Gdk

def get_monitors():
    display = Gdk.Display.get_default()
    n_monitors = display.get_n_monitors()
    monitors = []
    
    for i in range(n_monitors):
        monitor = display.get_monitor(i)
        geometry = monitor.get_geometry()
        width_mm = monitor.get_width_mm()
        height_mm = monitor.get_height_mm()
        name = monitor.get_model()
        is_primary = monitor.is_primary()

        monitors.append({
            'x': geometry.x,
            'y': geometry.y,
            'width': geometry.width,
            'height': geometry.height,
            'width_mm': width_mm,
            'height_mm': height_mm,
            'name': name,
            'is_primary': is_primary,
        })
    
    return monitors

monitors = get_monitors()
for monitor in monitors:
    print(monitor)

```

Reply:
I will do it, and after test it i will push. Thanks.